package.json

```
@@ -17,28 +17,28 @@
 	"browser": "main.js",
 	"scripts": {
 		"start": "npx serve ./demos/local",
-		"build": "npm_config_yes=true npx \"origami-build-tools@prerelease\" install && npm_config_yes=true npx \"origami-build-tools@prerelease\" demo"
+		"build": "npm_config_yes=true npx \"origami-build-tools@11\" install && npm_config_yes=true npx \"origami-build-tools@11\" demo"
 	},
 	"devDependencies": {
 		"eslint": "^7.24.0",
 		"eslint-config-origami-component": "^2.1.1",
 		"remark": "^13.0.0",
 		"remark-lint": "^7.0.1",
-		"remark-preset-lint-origami-component": "prerelease",
+		"remark-preset-lint-origami-component": "^2.0.0",
 		"stylelint": "^13.12.0",
 		"stylelint-config-origami-component": "^1.0.4"
 	},
 	"peerDependencies": {
-		"@financial-times/o-brand": "prerelease",
-		"@financial-times/o-colors": "prerelease",
-		"@financial-times/o-fonts": "prerelease",
-		"@financial-times/o-grid": "prerelease",
-		"@financial-times/o-normalise": "prerelease",
-		"@financial-times/o-quote": "prerelease",
-		"@financial-times/o-spacing": "prerelease",
-		"@financial-times/o-typography": "prerelease",
-		"@financial-times/o-utils": "prerelease",
-		"@financial-times/o-visual-effects": "prerelease"
+		"@financial-times/o-brand": "^4.0.2",
+		"@financial-times/o-colors": "^6.0.1",
+		"@financial-times/o-fonts": "^4.5.0",
```

I found out that o-fonts didn't publish correctly which is why this is using ^4.
I've published the correct version, which is 5.0.0
```suggestion
		"@financial-times/o-fonts": "^5.0.0",
```